CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ cfncluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the cfncluster cookbook.
 
+1.5.0
+-----
+
+Major new features/updates: 
+  - Added parameter to specify custom cfncluster-node package
+
+Bug fixes/minor improvements:
+  - Fix eth0 interface renaming
+  - fix poise-python dependecy issue
+  - Reboot after package update
```

Comment:
this was added in the kitchen test execution to align with the behaviour done during ami creation (packer)
We are not rebooting the instance at "cfncluster creation time".
I think we should remove this